primer/src/Primer/Builtins/Picture.hs

```diff
@@ -0,0 +1 @@
+module Primer.Builtins.Picture () where
```

Comment:
Did you mean to commit this empty module? (You have referred to it in `primer.cabal` also). Maybe add a comment explaining the purpose?